mlflow/utils/autologging_utils.py

```diff
@@ -381,7 +415,15 @@ def autologging_is_disabled(flavor_name):
 
     :param flavor_name: An autologging integration flavor name.
     """
-    return get_autologging_config(flavor_name, "disable", True)
+    explicit_disabled = get_autologging_config(flavor_name, "disable", False)
```

Comment:
@harupy 
I change this line 
`get_autologging_config(flavor_name, "disable", True)`
to
`get_autologging_config(flavor_name, "disable", False)`
is it right ?

Reply:
why did you need this change?

Reply:
Because in every package autolog definition the disable argument default value is False, like:
https://github.com/mlflow/mlflow/blob/2d3ac48b01da021d220fb1bf4450c70976f3b5d7/mlflow/sklearn/__init__.py#L536

CC @dbczumar

Reply:
@WeichenXu123 we should keep the previous construction of the line, i.e.:

```
is_autologging_disabled_for_flavor = get_autologging_config(flavor_name, "disable", True)
```

The reason is that we need to capture explicit *and* implicit disablement. For example, in the case of scikit-learn, if the user hasn't called the `mlflow.autolog()` or `mlflow.sklearn.autolog()` APIs, then `autologging_is_disabled("sklearn")` should return `True`. However, in this case, it seems to return `False`:

```
In [1]: import mlflow

In [2]: from mlflow.utils.autologging_utils import autologging_is_disabled

In [3]: autologging_is_disabled("sklearn")
Out[3]: False
```

Can we fix this and add test coverage?

Reply:
There's already a test cover this case. But, strange, the test is skipped.
https://github.com/mlflow/mlflow/blob/2d3ac48b01da021d220fb1bf4450c70976f3b5d7/tests/autologging/test_autologging_utils.py#L560

It is skipped because of this mark:
https://github.com/mlflow/mlflow/blob/2d3ac48b01da021d220fb1bf4450c70976f3b5d7/tests/autologging/test_autologging_utils.py#L32

CC @harupy Could you help check this ?

Reply:
hmm, looks like files under `tests/autologging` are not executed.